// Try Catch

const numero1 = 20;
const numero3 = 30;

console.log(numero1);
// Solo se utiliza en casos realmente nesesarios
try{
    console.log(numero2);
}catch(error){
    console.log(error);
}

console.log(numero3);